about.html

```diff
@@ -0,0 +1,188 @@
+                    </p>
+                </div>
+        
+            </div>
+
+            <div class="row pb-5 align-items-center text-center justify-content-center">
+                <div class=" col-md-4 summit-logo p-2">
+                   <img id="summit" src="assets/images/logo/Tech.png" alt="techlife logo" class="img-fluid">
+                </div> 
+            </div>
+        </div>
+        <hr>
+    </section>
+
+    <!-- feature speakers -->
+    <section id="past-summit">
+        <div class="container-lg">
+            <div class="text-center">
+                <h2 class="summit-heading pt-4 pb-2">See the Past TechLife Global Summits</h2>
+                <div class="line"></div>
+                <p class="past-p p-3">Take a look at the past two TechLife Global Summits which took place in Nairobi Kenya </p>
+            </div>
+            <div class="row align-items-center justify-content-center pb-5 pt-2 g-1">
+
+                <div class="col-md-4 me-2 text-center">
+                    <div class="img-fluid">
+                        <div class="img-container">
+                            <img src="assets/background/past-summit-1.png" alt="Techlife 2021 summit" class="img-fluid">
+
+                            <div class="content pt-5">
+                              <h2 class="img-header">2022</h2>
+                              <p class="img-text">Tibika Innovative Global Summit in 2022</p>
+                            </div>
+                        </div>
+                    </div>              
+                </div>  
+
+                <div class="col-md-4 me-2 text-center">
+                    <div class="img-fluid">
+                        <div class="img-container">
+                            <img src="assets/background/past-summit-2.png" alt="Techlife 2021 summit" class="img-fluid">
+
+                            <div class="content pt-5">
+                              <h2 class="img-header">2023</h2>
+                              <p class="img-text">Tibika Innovative Global Summit in 2023</p>
+                            </div>
+                        </div>
+                    </div>              
+                </div>  
+
+                
+                 
+            </div>
+        </div>
+    </section>
+
+    <!-- footer  -->
+
+    <section id="footer">
+        <div class="container-fluid about-footer py-5">
+                <div class="col-md-8 offset-md-2">
+                    <div class="footer-p-logo">
+                        <div class="footer-logo">
+                            <img src="assets/images/logo/techlife.png" alt="mozilla" class="img-fluid logo">
+                        </div>
+                        <p class="about-footer-p footer-p">2023 tibika Korea. Some Rights Reserved.</p>
+                    </div>
+                </div> 
+        </div>
+    </section>
```

Comment:
- [ ] For the about page footer section, the partner and the footer sectios are visible on the mobile. Only the footer that's visible on the desktop. Kindly readdress that.
- [ ] Also, you will need to increase the left and right margin of the two images in order to match the guiding template. Take a closer look at the both screenshots, compare the margin for the two images and walk your design alongside the guiding template.

### Your Design (about page) Mobile
![your-about-footer](https://github.com/samuelkinyanjui/Capstone-Project_sam/assets/26597432/c3b0a026-eeb3-4c35-80e9-f14aa045a969)

### Template Design (about page) Mobile
![temp-about-footer](https://github.com/samuelkinyanjui/Capstone-Project_sam/assets/26597432/42c07afa-d2b9-4768-9cb2-e3ff126bf20c)